geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreFactoryTest.scala

```diff
@@ -76,8 +78,8 @@ class AccumuloDataStoreFactoryTest extends Specification {
         AccumuloDataStoreParams.PasswordParam.key   -> MiniCluster.Users.root.password,
         AccumuloDataStoreParams.CatalogParam.key    -> catalog
       ).asJava
-      AccumuloDataStoreFactory.canProcess(params) must beFalse
-      DataStoreFinder.getDataStore(params) must beNull
+      AccumuloDataStoreFactory.canProcess(params) must beTrue
+      DataStoreFinder.getDataStore(params) must throwAn[IOException]
```

Comment:
Ok, this is an interesting change.  If we know that we'd thrown an exception, does it make sense that canProcess should be true?

Reply:
see above